library/Icingadb/Hook/ActionsHook/ObjectActionsHook.php

```diff
@@ -40,11 +40,11 @@ final public static function loadActions(Model $object): HtmlElement
     {
         switch (true) {
             case $object instanceof Host:
-                /** @var HostActionsHook $hook */
+                /** @var HostActionsHook $hookName */
```

Comment:
phpstorm gets this right. This previously meant $hook for a reason, as that's what's used by the loop below.